phoenicis-dist/src/resources/Phoenicis.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Encoding=UTF-8
+Name=Phoenicis
+GenericName=Game
+Comment=Phoenicis
+Type=Application
+Exec=phoenicis %F
+Icon=phoenicis
+Categories=Game;
```

Comment:
I don't like the category.
I mean `Phoenicis` is not really a game or a game management application it is much more generic.

Reply:
Yep, any other suggestion?

Reply:
I think @plata had some additional categories in our old `.desktop` file.
I don't remember in detail which they were

Reply:
You can put it in multiple category. Utility and games, for examples.

Reply:
Yes, the previous control file had "Game;Utility" which is what I used now for the flatpak as well (see https://github.com/PhoenicisOrg/phoenicis/blob/master/phoenicis-dist/src/flatpak/org.phoenicis.playonlinux.appdata.xml).
Just realized that there is a mistake... The appdata.xml and the .desktop are using different categories. If we decide on the categories here, I can update that as well.